src/telegram.toit

```diff
@@ -161,7 +161,12 @@ class Client:
   /**
   Sends a message to the given chat.
```

Comment:
Add documentation.

Something like
```
If $disable_web_page_preview is true, then link previews are disabled.
If $disable_notification is true, then sends the message silently and users will receive a notification with no sound.
```
Use the Telegram API documentation for help writing the documentation.